docs/nbext/commit.md

```diff
@@ -1 +1,54 @@
 # Use Extension to Commit
+
+Inorder to use `jovian` extension you will need to first install Jovian on your system. Please visit [Installation](../user-guide/01-install.md) page before reading further.
+
+
+#### Using Jovian Extension
+
+Once you have sucessfully installed Jovian, a new button `/j Commit` will appear on the tool bar. When using `/j Commit` button for first time you'll be asked to provide an API key.
+
+                          
+<img src="https://i.imgur.com/oNSowtY.png" class="screenshot">
+
+ You can get the API key at [Jovian](https://jvn.io). You will need to sign in using your github or google account. If you do not have either of this accounts you will need to sign up. Once you log in, just click on `API key` button, and the key will be copied to the clipboard.
+
+<img src="https://i.imgur.com/taLLUVd.png" class="screenshot">
+
+#### Valid API key
+If the key is valid you will be notified with the following alert.
+
+<img src="https://i.imgur.com/UHvSihx.png" class="screenshot">
+
+#### Error with API key
+IF the entered API key is invalid you will get follwwoing error.
+
+<img src="https://i.imgur.com/9WaVkTR.png" class="screenshot">
+
+#### Sucessful Commit
+Once the API key has been validated, you can start commiting to Jovian by pressing `/j Commit` button. Once the document has been commited sucessfully you will be prompted with the following message. 
```

Comment:
use `clicking` instead of `pressing`.

Reply:
Add the information, saying you can visit the link to view your notebook on Jovian and they can copy the link with that copy button in the modal which they can use to share the notebook.